packages/ui/src/components/va-time-picker/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -5,26 +5,26 @@
     @keydown.down.stop.prevent="makeActiveNext()"
     @keydown.space.stop.prevent="makeActiveNext(5)"
     @keydown.up.stop.prevent="makeActivePrev()"
+    @scroll="onScroll"
     ref="rootElement"
   >
-    <div class="va-time-picker-cell va-time-picker-cell--fake" />
     <div
       v-for="(item, index) in items" :key="item"
-      :class="{
-        'va-time-picker-cell': true,
-        'va-time-picker-cell--active': index === $props.activeItemIndex
-      }"
+      class="va-time-picker-cell"
+      :class="{ 'va-time-picker-cell--active': index === $props.activeItemIndex }"
+      :data-index="index"
+      :data-item="item"
       @click="onCellClick(index)"
     >
       <slot name="cell" v-bind="{ item, index, activeItemIndex, items, formattedItem: formatCell(item) }">
         {{ formatCell(item) }}
       </slot>
     </div>
-    <div class="va-time-picker-cell va-time-picker-cell--fake" />
   </div>
 </template>
 
 <script lang="ts">
+import debounce from 'lodash/debounce'
```

Comment:
```suggestion
import debounce from 'lodash/debounce.js'
```

Sorry 😄, hope we will fix that requirement ASAP.